src/Lesson1/Task1/Tools.java

```diff
@@ -0,0 +1,10 @@
+package Lesson1.Task1;
+
+public class Tools {
```

Comment:
Tools - saying nothing about what this class will do, so please, rename it. Most probably this class will be logger, or something like this. Please, read single responsibility principle.